src/app/core/export/download-service/download.service.spec.ts

```diff
@@ -99,15 +100,19 @@ describe("DownloadService", () => {
     testEntity.dateProperty = new Date(testDate);
     testEntity.boolProperty = true;
 
+    const objectToArray = [
+      testEntity.enumProperty.label,
+      testEntity.dateProperty.toISOString(),
+      testEntity.boolProperty.toString(),
+      ];
+      
     const csvExport = await service.createCsv([testEntity]);
 
     const rows = csvExport.split(DownloadService.SEPARATOR_ROW);
     expect(rows).toHaveSize(1 + 1); // includes 1 header line
-    const columnValues = rows[1].split(DownloadService.SEPARATOR_COL);
-    expect(columnValues).toHaveSize(3 + 1); // Properties + _id
-    expect(columnValues).toContain('"' + testEnumValue.label + '"');
-    expect(columnValues).toContain('"' + testDate + '"');
-    expect(columnValues).toContain('"true"');
+    expect(objectToArray).toContain(testEnumValue.label);
+    expect(objectToArray).toContain(new Date(testDate).toISOString());
+    expect(objectToArray).toContain('true');
```

Comment:
You are not testing the result of the `createCsv` method anymore now. objectToArray seems to only be something manually set up for the test?